src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Contracts/Payments/ListPaymentsResponse.cs

```diff
@@ -0,0 +1,6 @@
+namespace TicketManager.PaymentService.Contracts.Payments;
+
+public class ListPaymentsResponse
+{
+    public List<PaymentDto> Payments { get; set; }
+}
```

Comment:
Add new line.

Reply:
Configure your `IDE` to do it automatically or @michal12334 will haunt you for every single missing new line

Reply:
File without new line at the end is not correct posix file. Editors like vim and nano don't even allow you to not add new line at the end. Also github add red sign that looks like error if you don't add it.

Reply:
Addressed in `4ce71b91` and configured `IDE`.